src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,251 @@
+            }
+            if(Integer.parseInt(words[1].trim())==1) {
+                tasks.get(totalTasks).markAsDone();
+            }
+            totalTasks++;
+        }
+    }
+
+    private static void saveTaskList() throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        for(int i = 0; i< totalTasks; ++i) {
+            fw.write(tasks.get(i).storeString() + "\n");
+        }
+        fw.close();
+    }
+
+    private static void addTodo(String task) {
+        tasks.add(new Todo(task));
+    }
+
+    private static void addEvent(String task) {
+        String[] words = task.split("/at");
+        tasks.add(new Event(words[0], words[1]));
+    }
+
+    private static void addDeadline(String task) {
+        String[] words = task.split("/by");
+        tasks.add(new Deadline(words[0], words[1]));
+    }
+
+    private static void updateDone(int updateNumber) {
+        tasks.get(updateNumber).markAsDone();
+    }
+
+    private static void updateDelete(int deleteNumber) {
+        tasks.remove(deleteNumber);
+    }
+
+    private static void printList() {
+        if(totalTasks==0) {
+            System.out.println("List is empty.");
+            return;
+        }
+        System.out.println("Here are the tasks in your list:");
+        for(int taskNumber = 0; taskNumber < totalTasks; taskNumber++) {
+            System.out.println(taskNumber + 1 + "." + tasks.get(taskNumber));
+        }
+    }
+
+    private static void printDone(int updateNumber) {
+        System.out.println("Nice! I've marked this task as done: ");
+        System.out.println(tasks.get(updateNumber));
+    }
+
+    private static void printDelete(int deleteNumber) {
+        System.out.println("Noted. I've removed this task: ");
+        System.out.println(tasks.get(deleteNumber));
+        totalTasks -= 1;
+        System.out.println((totalTasks==1)? "Now you have 1 task in the list." : "Now you have " + totalTasks + " tasks in the list.");
+    }
+
+    private static void printUpdate() {
+        System.out.println("Got it. I've added this task: ");
+        System.out.println(tasks.get(totalTasks));
+        totalTasks++;
+        System.out.println((totalTasks==1)? "Now you have 1 task in the list." : "Now you have " + totalTasks + " tasks in the list.");
+    }
+
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println(logo);
+        loadFile();
+        System.out.println("Hello! I'm Duke.");
+        System.out.println("What can I do for you?");
+        String command;
+        String lowerCaseCommand;
+        do {
+            Scanner in = new Scanner(System.in);
+            command = in.nextLine();
+
+            // remove all unnecessary spaces in input
```

Comment:
The general format for in-line comment is `// Remove all ... in input.` You have to capitalize at the stand and put a `.`  at the end of the comment.